### PR TITLE
fix(radarr): Increase preferred and maximum Radarr quality definition sizes

### PR DESCRIPTION
**Pull request by yammes08**
# Pull Request

## Purpose

To maintain compatibility with Radarr after it merges an increase to the quality definition ranges - namely that `2000` is now 'Unlimited' where `400` was the previous value.

## Approach

In the Radarr quality definitions JSON
- Change `399` to `1999` for preferred quality
- Change `400` to `2000` for max quality

## Open Questions and Pre-Merge TODOs

None

## Requirements

- [x] These changes meet the standards for [contributing](https://github.com/TRaSH-Guides/Guides/blob/master/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/TRaSH-Guides/Guides/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Comment by yammes08** [comment]
Can be merged once these changes land in Radarr Master.

**Review by bakerboy448 — APPROVED** [review]
radarr master released
https://github.com/Radarr/Radarr/releases/tag/v5.9.1.9070

**Pull request merged**
